packages/@aws-cdk/aws-redshift-alpha/test/cluster.test.ts

```diff
@@ -779,6 +779,31 @@ describe('default IAM role', () => {
 });
 
 describe('IAM role', () => {
+
+  test('cluster instantiated with a default grantable IAM Role', () => {
```

Comment:
I think we need unit tests for:
* A `Cluster` with `serviceRole` specified via props
* A `Cluster` with `serviceRole` and `roles` specified via props